follower/consensus_follower.go

```diff
@@ -211,10 +220,17 @@ func runAccessNode(ctx context.Context, anb *access.UnstakedAccessNodeBuilder) {
 		anb.Logger.Info().Msg("Access node startup complete")
 	case <-ctx.Done():
 		anb.Logger.Info().Msg("Access node startup aborted")
+	case err := <-anb.Errors():
+		cf.errorManager.ThrowError(err)
+	}
+
+	select {
+	case <-ctx.Done():
+		anb.Logger.Info().Msg("Access node shutting down")
+	case err := <-anb.Errors():
+		cf.errorManager.ThrowError(err)
 	}
 
-	<-ctx.Done()
-	anb.Logger.Info().Msg("Access node shutting down")
 	<-anb.Done()
```

Comment:
Technically error could occure during shutdown too, so maybe work wrapping this inside select statement as well, similar to what you did above